planetiler-core/src/main/java/com/onthegomap/planetiler/expression/Expression.java

```diff
@@ -31,6 +34,19 @@ public interface Expression {
   String POLYGON_TYPE = "polygon";
   String RELATION_MEMBER_TYPE = "relation_member";
 
+  String STRING_DATATYPE = "string";
+  String BOOLEAN_DATATYPE = "boolean";
+  String DIRECTION_DATATYPE = "direction";
+  String LONG_DATATYPE = "long";
+
+  static final Map<String, BiFunction<SourceFeature, String, Object>> dataTypeGetter =
+    Collections.unmodifiableMap(Map.of(
+      STRING_DATATYPE, SourceFeature::getString,
+      BOOLEAN_DATATYPE, SourceFeature::getBoolean,
+      DIRECTION_DATATYPE, SourceFeature::getDirection,
+      LONG_DATATYPE, SourceFeature::getLong
+    ));
+
```

Comment:
I've changed this to take in the BiFunction as a parameter.  This will allow us to not only specify which getter, but also load on any other formatting we might light to add as a pre-processing step, which I anticipate will be useful to the declarative schema effort.